qml/FinishedPage.qml

```diff
@@ -65,6 +68,28 @@ BasePage {
             }
         }
 
+        Item {
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: Units.gu(2)
+        }
```

Comment:
Is there a specific reason to use this filler Item, instead of using anchors.topMargin in the next Text item ?